docs/_documentations/mdt-eclipse-usingadifferenttemplate.md

```diff
@@ -10,15 +10,10 @@ order: 8
 parent: usingcodewind
 ---
 
-# Using a template repository of your own
+# Configuring template repositories
 
-When you create a new project, you can choose from the default set of templates available, or you can choose from a template repository of your own. To add a new repository, edit the `.config/repository_list.json` file within your `codewind-workspace`. The file is a json file that contains an array of objects with two fields, the URL of the repository and a description:
+When you create a new project, you can choose from the default set of templates available, or you can choose from a template repository of your own. Use Codewind to manage templates. Templates can make application development easier by providing a structure and boilerplate code to help get a new project started.
 
-```
-{
-  "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
-  "description": "Standard Codewind templates."
-}
-```
-
-After you have edited and saved this file, restart Codewind.
+- To open the Template Source Manager, right-click **Projects (Local)**.
+- Click **Add New** to add a repository source to the table.
+- Enable or disable a repository type to view the repositories of that type in the wizard.
```

Comment:
In the dialog, use the check boxes to enable or disable repositories.  Use the **Add** button to a new repository.  The **Remove** button can be used to remove non-default repositories.